### PR TITLE
Angostura connection module

### PR DESCRIPTION
**Pull request by dieko95**
# About this PR

-  `AngosturaConnection.create_query` is a generic method to query data from Angostura's table `sinluz_rawtweets`.
-  To query Angostura it is necessary to have the `angostura_connection.json`key and add it as an environment variable. Please ask C4V's Sambil team for this key.

# Running these changes locally 

## Add `angostura_connection` key.

`angostura_connection.json` is the json key to query angostura. Please request it to the Sambil team. 

1. Copy the json key to the project's root folder. 
2. Run the folllowing command to add the json key as an environment variable:

```
export ANGOSTURA_SERVICE_ACCOUNT="/path/to/angostura_connection.json"
```

3. Activate poetry shell and run tests: `nox -r -s`

##  Query Angostura
```
    test = AngosturaLoader()
    df = test.create_query(
        "SELECT * FROM `event-pipeline.angostura.sinluz_rawtweets` LIMIT 1"
    )
    df.info()

``` 

To fix:
- The json key needs to be added to github secrets. This commit does not
includes the json key file due to security concerns.

### CODE REVIEW

**Review by marianelamin — APPROVED** [review]
LGTM

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -19,9 +19,9 @@ jobs:
           - name: 'Python 3.6.10'
             os: ubuntu-18.04
             python-version: '3.6.10'
-          - name: 'Python 3.7.7'
+          - name: 'Python 3.7.9'
             os: ubuntu-18.04
-            python-version: '3.7.7'
+            python-version: '3.7.9'
```
Comment (dieko95) [inline .github/workflows/main.yml:24]: @Edilmo I have changed the python version because the test build was failing.

Comment (Edilmo) [reply .github/workflows/main.yml:24]: Sounds good to me. I had problems with this as well

**Review by Edilmo — APPROVED** [review]

**Comment by dieko95** [comment]
I will merge this PR and will open another PR to fix Github actions. The tests are passing locally. 

CC: @Edilmo

**Pull request merged**